### PR TITLE
Confirm congrate

### PR DESCRIPTION
**Pull request by yaqootturman**

### CODE REVIEW

File: client/src/components/PledgePage/index.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    const { data } = this.props.location.state
+    const { pledgeInfo, pledgeInstructions, pledgeProsCons, pledgeResources, pledgeReferences } = this.state
+
+    return (
+      <>
+        {
+          !pledgeInfo.length ? <p>Loading...</p> :
+            <>
+              <div className="top-info">
+                <img className="top-info__img" src={pledgeInfo[0].img} />
```
Comment (ishak52) [inline client/src/components/PledgePage/index.js]: `img` elements must have an `alt` prop

File: client/src/components/HomePage/index.js [hunk 2]
```diff
@@ -1,36 +1,39 @@
-
-import React, { Component, useContext } from 'react'
-// import UserPledges from '../UserPledges'
-// import Footer from '../Footer'
-// import axios from 'axios'
+import React, { Component } from 'react'
+import UserPledges from '../UserPledges'
+import Footer from '../Footer'
+import axios from 'axios'
```
Comment (ishak52) [inline client/src/components/HomePage/index.js]: 'axios' is defined but never used

File: client/src/components/PledgItem/PledgeItem.js [hunk 3]
```diff
@@ -1,13 +1,10 @@
...
-  const { element } = props
-  //should git history to be 
-  // const { element, history } = props
-
+  const { element, history } = props
   return (
     <div className="pledge-div">
-      <img onClick={() => { console.log("history.push('')") }} alt={element.title} className="pledge-div__img" src={element.img} />
+      <img onClick={() => { history.push(`/action-category/pledge/${element.title}`, { pledge_id: element.pledge_id }) }} alt={element.title} className="pledge-div__img" src={element.img} />
       <p className="pledge-div__title">{element.title}</p>
```
Comment (ishak52) [inline client/src/components/PledgItem/PledgeItem.js]: this element <p className="pledge-div__title">{element.title}</p>
needing to fix it's style it's not presented in the right way in the page, fix it's style

**Comment by ishak52** [comment]
Fix travis

**Comment by ishak52** [comment]
write comments on your code when it's necessary

**Comment by ishak52** [comment]
I can't redirect from my pledges page to a specific pledge page

File: client/src/components/PledgePage/index.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        pledgeResources: pledgeInfoPage[3],
+        pledgeReferences: pledgeInfoPage[4]
+      })
+
+    }).catch(err => { console.log("0000", err) });
+
+  }
+
+  render() {
+    const { data } = this.props.location.state
```
[the remark this answers is not in the document]
Comment (yaqootturman) [reply client/src/components/PledgePage/index.js]: This depends on what we recieve and i changed it to pledge_id cause i sent it from homePledge page and categoryPledges page a pledge_id

**Pull request merged**